app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -596,10 +596,10 @@
             .header-slot,
             .viewlet-manager-navigation,
             .viewlet-manager-footer {
-                @include flex-basis(auto);
```

Comment:
They are used elsewhere, and actually it looks like there might be some problems in the machine view when there are a lot of machines. I'll add a card.

The reason it works is now we're explicitly defining the flex grow, shrink and basis. Chrome was having some issues applying the flex model correctly without those explicit properties.